### PR TITLE
[Scheduler] Document hashed Cron expression

### PR DESCRIPTION
**Pull request by alamirault**
Follow #19440 

Try document https://github.com/symfony/symfony/pull/49792

I copied a lot https://github.com/zenstruck/schedule-bundle/blob/1.x/doc/define-tasks.md#hashed-cron-expression

### CODE REVIEW

File: scheduler.rst [hunk 1]
```diff
@@ -168,22 +175,67 @@ Then, define the trigger date/time using the same syntax as the
...
+* ``@daily``, ``@midnight`` - Run once a day, midnight - ``0 0 * * *``
+* ``@hourly`` - Run once an hour, first minute - ``0 * * * *``
 
 For example::
 
-    RecurringMessage::cron('#daily', new Message());
+    RecurringMessage::cron('@daily', new Message());
+
+Hashed Cron Expression
+~~~~~~~~~~~~~~~~~~~~~~
```
Comment (alamirault) [inline scheduler.rst:196]: I want to make a sub section, which char should I use ?

File: scheduler.rst [hunk 2]
```diff
@@ -168,22 +175,67 @@ Then, define the trigger date/time using the same syntax as the
...
+
+If you have many trigger scheduled at same time (for example, at midnight, ``0 0 * * *``)
+this could create a very long running schedule right at this time.
+This may cause an issue if a task has a memory leak.
```
Comment (alamirault) [inline scheduler.rst:200]: I don't like this sentence. Maybe we should talk about real use case.
If I remember fabpot talk about email campaign

Comment (94noni) [reply scheduler.rst:200]: what about writing bottle neck like lot of I/O, memory spike, etc etc?

File: scheduler.rst [hunk 3]
```diff
@@ -168,22 +175,67 @@ Then, define the trigger date/time using the same syntax as the
...
+
+You can add a ``#``(for hash) symbol in expression to generate random value. The value
+is deterministic based on the message. This means that while the value is random, it is
+predictable and consistent. A message with string representation ``my task``
+and a defined frequency of ``# # * * *`` will have a calculated frequency
```
Comment (94noni) [inline scheduler.rst]: does "calculated" means "idempotent" ?

File: scheduler.rst [hunk 4]
```diff
@@ -168,22 +175,67 @@ Then, define the trigger date/time using the same syntax as the
...
 For example::
 
-    RecurringMessage::cron('#daily', new Message());
+    RecurringMessage::cron('@daily', new Message());
+
+Hashed Cron Expression
+~~~~~~~~~~~~~~~~~~~~~~
+
+If you have many trigger scheduled at same time (for example, at midnight, ``0 0 * * *``)
+this could create a very long running schedule right at this time.
```
Comment (94noni) [inline scheduler.rst]:
```suggestion
this will create a very long running schedules list right at this time.
```

**Review by 94noni — COMMENTED** [review]
to me its a nice addition to this component 👍🏻

File: scheduler.rst (hunk 1 above)
Comment (javiereguiluz) [reply scheduler.rst:196]: We use the `...........` separator for those sections. I did this while merging your PR.

**Comment by javiereguiluz** [comment]
Merged! Thanks Antoine.

**Pull request merged**